macosx/PrefsController.mm

```diff
@@ -103,6 +104,7 @@ @interface PrefsController ()<NSWindowRestoration>
 @property(nonatomic, readonly) NSMutableArray<NSString*>* fRPCWhitelistArray;
 @property(nonatomic) IBOutlet NSSegmentedControl* fRPCAddRemoveControl;
 @property(nonatomic, copy) NSString* fRPCPassword;
+@property(nonatomic) DefaultAppHelper* fDefaultAppHelper;
```

Comment:
It does not change so I suppose we can make it `readonly`

Reply:
Fixed